### ISSUE TITLE
Blocked by GFW (Cryptocurrency Exchange)

### ISSUE THREAD

**Issue by Timolol2012** [gfwlist/gfwlist#1002]
gate.io
bcex.ca
big.one
jex.com
cex.com
exx.com
aex.com
yobit.net
bjex.com
bit-z.com
tidex.com
lbank.info
bibox.com
coinw.com
allcoin.com
coinex.com
btctrade.im
topbtc.com
kucoin.com
leekico.com
chaoex.com
shapeshift.io
coinegg.com
coolcoin.com
ico-china.info
ebtcbank.com
etherdelta.com
bitcoinworld.com

[notepad.txt](https://github.com/gfwlist/gfwlist/files/1910365/notepad.txt)

**Comment by cicku** [comment]
Duplicate of #947
